core/common/src/AtmosphericScattering.ts

```diff
@@ -0,0 +1,170 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module DisplayStyles
+ */
+import { JsonUtils } from "@itwin/core-bentley";
+import { ColorDef, ColorDefProps } from "./ColorDef";
+import { RgbColorProps } from "./RgbColor";
+
+/**
+ * @public
+ */
+export namespace AtmosphericScattering {
+
+  export class Wavelengths implements RgbColorProps {
```

Comment:
This class appears to add nothing over [RgbColor](https://www.itwinjs.org/reference/core-common/symbology/rgbcolor/).

Reply:
RgbColor limits values in the range [0,255]. I need to accept any positive value.

Reply:
Then you should not be using RgbColorProps and you should document this.